run.sh

```diff
@@ -2,6 +2,46 @@
 
 set -o errexit
 
+usage() {
+    echo "deploy                                         -- Deploy SUSE Containerized OpenStack."
+    echo "update_openstack                               -- Update OpenStack deployment."
+    echo "add_openstack_compute                          -- Add OpenStack compute node. User need to provide compute node name in inventory file."
+    echo "remove_openstack_compute <compute-node-name>   -- Remove OpenStack compute node. User need to provide compute node name with option."
```

Comment:
Can we request confirmation before removing compute or remove deployment?

Reply:
For cli I think non-interactive is better because they can automate if required.  From UI they can have that option ?